src/app/components/ui/form/form.component.html

```diff
@@ -75,7 +75,7 @@
         <!-- Numeric -->
         <ng-container *ngSwitchCase="FieldType.Numeric">
             <input
-                type="text"
+                type="number"
```

Comment:
`type="text"` was purposefully, because styling of numeric controls on different browsers/platforms can be different. If you want this type, I'll check it later

Reply:
I need to send Integer/Numeric fields as numbers, not strings which input type text did. 

Specific input type can be anything you think is right.